src/pipeline_eval.py

```diff
@@ -45,7 +47,7 @@ def create_pipeline(  # noqa: PLR0913
     )
```

Comment:
Add `"api_keys": api_keys` in arguments. Same in pipeline_index.

Reply:
Let's add this as a separate change.